connectors/src/connectors/zendesk/lib/zendesk_api.ts

```diff
@@ -165,3 +168,81 @@ export async function fetchZendeskArticlesInCategory({
 
   return response;
 }
+
+export async function fetchZendeskTicketsInBrand({
+  subdomain,
```

Comment:
`nit`: rename this to `brandSubdomain` to make it more obvious that this is why this function only acts on a brand